microsoft-edge/progressive-web-apps-chromium/how-to/index.md

```diff
@@ -138,63 +164,208 @@ Your VS Code project should now look somewhat like this:
 
 
 <!-- ====================================================================== -->
-## Step 3 - Add a Service Worker
+## Step 4 - Continue building the user interface of your app
 
-Service workers are the key technology behind PWAs.  Service workers enable scenarios that were previously limited to native apps, including:
+Now that your app has a web app manifest file, and a start page, it is time to build out the main app functionality.
 
-*  Offline support.
-*  Advanced caching.
-*  Running background tasks.
+In this step of the tutorial, we will create a basic temperature unit conversion app.
```

Comment:
```suggestion
In this step of the tutorial, we'll create a basic temperature unit conversion app.
```
Acrolinx would suggest contraction.  Aim for transparent/natural tone, to decide.